conda-forge.yml

```diff
@@ -5,4 +5,6 @@ appveyor:
   secure:
     BINSTAR_TOKEN: tumuXLL8PU75WMnRDemRy02ruEq2RpNxeK3dz0MjFssnosPm2v4EFjfNB4PTotA1
 provider:
+  linux_aarch64: azure
+  linux_ppc64le: azure
```

Comment:
Why use azure?

Reply:
What to use instead? 
```
  linux_aarch64: native
  linux_ppc64le: default
```

Is this about build speed and distributing the CI load to different providers?

Reply:
Use,
```
  linux_aarch64: default
  linux_ppc64le: default
```

azure runs on x86_64 hardware and uses QEMU. drone and travis-ci runs on native hardware and therefore will have no bugs due to QEMU. For example, in numpy, there were tests failures due to QEMU.